scripts/create-agent.ts

```diff
@@ -282,42 +287,48 @@ GITHUB_TOKEN=${envVars.GITHUB_TOKEN || process.env.GITHUB_TOKEN || ''}`;
         { tag: 'PYTHON', command: 'sudo apt install -y python3.12 python3.12-venv python3-pip' },
         { tag: 'PY3_12', command: 'sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.12 1' },
         { tag: 'PY_ALIAS', command: 'sudo update-alternatives --install /usr/bin/python python /usr/bin/python3.12 1' },
+        { tag: 'SSM_INSTALL', command: 'sudo snap install amazon-ssm-agent --classic' },
+        { tag: 'SSM_START', command: 'sudo snap start amazon-ssm-agent' },
```

Comment:
I see that this Started successfully, but the health check is failing with the same error